datamade/erikson-edi
# another place to change subdomain language

From Veronica:

![0](https://user-images.githubusercontent.com/919583/72277094-ea753c00-35f6-11ea-9b87-506fe77b5489.png)

This one line change resolves this for subdomains.

###Testing instructions

Open up the map and select any subdomain. The 'meeting few/no developmental expectations' language should show up in the info window. Confirm that domains when selected are unaffected.
